Java shared project no longer using maven feed

seanmcilvenna opened the pull request:
* Shared is directly referenced as a project
* Root POM includes dependencies that are used by both measureeval and validation services
* Added repository for measure definitions to measureeval
* Added REST controller for measure definitions and tested that mongo connection works
* Dockerfiles updated to build both destination service and shared module at same time

seanmcilvenna committed:
* Shared is directly referenced as a project
* Root POM includes dependencies that are used by both measureeval and validation services
* Added repository for measure definitions to measureeval
* Added REST controller for measure definitions and tested that mongo connection works

smailliwcs commented on Java/measureeval/Dockerfile:
```diff
@@ -2,17 +2,14 @@ ARG SHARED_BOTW_FEED_TOKEN_ARG
 EXPOSE 8080
-CMD ["java", "-jar", "measureeval.jar"]
\ No newline at end of file
+CMD ["java", "-jar", "measureeval.jar", "--add-opens java.base/java.util=ALL-UNNAMED"]
```
Just curious ... is this something Mongo needs for serialization?

smailliwcs commented on Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/controllers/HealthController.java:
```diff
@@ -5,7 +5,7 @@
 
 @RestController
-@RequestMapping("/_health")
+@RequestMapping("/health")
```
Do we have an project-wide convention to put this at `/health` rather than `/_health`?  `SecurityHelper` seems to assume it's at `/_health`.  Regardless, should probably put the correct value in a constant and reference that where needed.

smailliwcs commented on Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/entities/MeasureDefinition.java:
```diff
@@ -0,0 +1,30 @@
+    private String measureName;
+
+    @Field("url")
+    private String measureUrl;
```
Any reason _not_ to have the Java field names match the Mongo property names?

seanmcilvenna committed:
Creating separate random ports for the two java services so that they can run at the same time without port conflicts

seanmcilvenna committed:
Changing port to 5135 to align with existing .NET service that will be replaced (should simplify deployment)
Changing MeasureDefinition.measureUrl to measureDefinitionUrl... It's not the URL of the measure, it's the URL to where to *get* the measure definition

smailliwcs reviewed (approved):
Dropped a few comments for further discussion, but gonna approve and merge so I can pull into my work.
